client/src/pages/Ping.js

```diff
@@ -1,51 +1,49 @@
-import { FormControl, TextField, Button, Typography } from "@material-ui/core";
+import { FormControl, TextField, Button, Typography } from '@material-ui/core';
```

Comment:
We don't need these components anymore (Ping, etc.)

Reply:
Will work on this next